api/authentication/authentication.js

```diff
@@ -0,0 +1,70 @@
+const express = require("express");
+const router  = express.Router();
+const Parse = require('parse/node')
+
+// Register route creates a user in the database, displays an error message
+// if the user cannot be created.
+router.post('/register', async (req, res) => {
+    let user = new Parse.User(req.body)
+  
+    try {
+        await user.signUp()
+        res.status(201)    
+        res.send({"user" : user})
+    } catch (error) {
```

Comment:
future idea: don't worry about this for this commit, but it would be interesting later to think about how we can handle and present errors in a standardized way

Reply:
Sounds good!!